app/scripts/services/utils.srv.js

```diff
@@ -105,6 +105,70 @@ angular.module('dashboard')
                 $log.error("Utils.openNewWin: ignored due to bad url " + aUrl);
             }
         };
+        /*
+         * @name dashboard.utils.stringIdForError
+         * @description Resolves a localized string id for an operation result code.
+         * @param {number} Result code to parse.
+         * @returns {string} String id.
+         */
+        Utils.stringIdForError = function stringIdForErrorFn(aNum) {
+            var res = "STR_ERR_OP_FAIL";
+            if (!angular.isNumber(aNum)) {
+                $log.error("Utils.stringIdForError: bad arg, not a number: " + JSON.stringify(aNum) +" type=" +typeof aNum);
+            }
+
+            switch (aNum) {
+                // Ahjo Meeting error codes
+                case CONST.MTGAPICODES.K1001.value:
+                    res = CONST.MTGAPICODES.K1001.strId;
+                    break;
+                case CONST.MTGAPICODES.K1002.value:
+                    res = CONST.MTGAPICODES.K1002.strId;
+                    break;
+                case CONST.MTGAPICODES.K1003.value:
+                    res = CONST.MTGAPICODES.K1003.strId;
+                    break;
+                case CONST.MTGAPICODES.K1004.value:
+                    res = CONST.MTGAPICODES.K1004.strId;
+                    break;
+                case CONST.MTGAPICODES.K1005.value:
+                    res = CONST.MTGAPICODES.K1005.strId;
+                    break;
+
+                default:
+                    $log.$log("Utils.stringIdForError: didn't find a string id for '" + aNum + "', defaulting to " + res);
```

Comment:
$log.$log is not a function. (In '$log.$log("Utils.stringIdForError: didn't find a string id for '" + aNum + "', defaulting to " + res)', '$log.$log' is undefined)